tests/integration/test_store_synapse.py

```diff
@@ -400,38 +400,31 @@ def test_mock_get_files_in_storage_dataset(
         with patch(
             "schematic.store.synapse.CONFIG", return_value=TEST_CONFIG
         ) as mock_config:
-            with patch.object(synapse_store, "syn") as mock_synapse_client:
```

Comment:
This is me misleading Gianna, but there are times when we want a connection with Synapse and there are times we don't.

This is probably a case where it is ok for there to a direct connection with Synapse to simplify the mocking - since you've included in your other `test/test_store.py` the mocked version.